src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo/index';
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {
+        comments.length > 0
+          // eslint-disable-next-line max-len
+          ? comments.map((comment) => <CommentInfo comment={comment} key={comment.id} />)
+          : <b data-cy="NoCommentsMessage">No comments yet</b>
+      }
```

Comment:
```suggestion
      {comments.length > 0
          // eslint-disable-next-line max-len
          ? comments.map((comment) => <CommentInfo comment={comment} key={comment.id} />)
          : <b data-cy="NoCommentsMessage">No comments yet</b>}
```
don't disable linter rules